src/System.Net.Security/tests/Scripts/setup-kdc.sh

```diff
@@ -0,0 +1,225 @@
+#!/usr/bin/env bash
+
+OS=`cat /etc/os-release | grep "^ID=" | sed 's/ID=//g' | sed 's/["]//g' | awk '{print $1}'`
+echo "Operating System: ${OS}"
+
+realm="TEST.COREFX.NET"
+
+principal1="TESTHOST/testfqdn.test.corefx.net"
+principal2="TESTHTTP"
+krb_user="krb_user"
+password="password"
+
+kadmin="kadmin.local"
+krb5kdc="krb5kdc"
+kdb5_util="kdb5_util"
+add_principal_cmd="add_principal -pw ${password}"
+
+krb_conf="krb5.conf"
+krb_conf_location="/etc/krb5.conf"
+keytabfile="/etc/krb5.keytab"
+
+PROGNAME=$(basename $0)
+usage()
+{
+	echo "Usage: ${PROGNAME} [-y|--yes] [-u|--uninstall]";
+}
+
+# Cleanup config files and uninstall KDC
+clean_up()
+{
+	echo "Stopping KDC.."
+	if pgrep krb5kdc 2> /dev/null; then sudo killall krb5kdc ; fi
```

Comment:
Rather than the script itself using ```sudo```, would it work to require the invoker of the script to use ```sudo``` to invoke it?  That would further prevent any concerns around the script going wild, as it wouldn't implicitly try to elevate.